src/XRP/default-xrp-client.ts

```diff
@@ -136,12 +141,14 @@ class DefaultXRPClient implements XRPClientDecorator {
    * @param drops A `BigInteger`, number or numeric string representing the number of drops to send.
    * @param destinationAddress A destination address to send the drops to.
    * @param sender The wallet that XRP will be sent from and which will sign the request.
+   * @param memos An optional list of memos to add to the transaction.
    * @returns A promise which resolves to a string representing the hash of the submitted transaction.
    */
   public async send(
     drops: BigInteger | number | string,
     destinationAddress: string,
     sender: Wallet,
+    memos?: XRPMemo[],
```

Comment:
I understand the XRP API is complex. Lets assume that XpringSDK is going to (imminently) expose a call that will let power users craft and send custom transactions. 

Would it make sense to make this API simpler? Most users will want a single memo as a string? Why not expose this parameter as `memo?: string?` rather than forcing external clients to understand what `XRPMemo` is? 

(If there's a legit answwer to this question I'm happy to hear it)

Reply:
The `Memo` is more than one field though (it's data, format and type) and it supports an arbitrarily sized array of memos as well. Maybe only attaching one memo is a reasonable assertion, but I honestly don't have enough context to know if saying that only specifying the `data` field is; I know we plan on specifying the format so that we know what to look for with our Open Payments ledger scanning. 

I think it maybe bite us to try to simplify something aggressively when we already have a use case that requires more fields than what you're suggesting, and I don't think we want to pass in multiple fields to describe one memo either.